controllers/controller/devworkspacerouting/devworkspacerouting_controller.go

```diff
@@ -41,32 +41,32 @@ var (
 	NoSolversEnabled = errors.New("reconciler does not define SolverGetter")
 )
 
-const workspaceRoutingFinalizer = "workspacerouting.controller.devfile.io"
+const devWorkspaceRoutingFinalizer = "devworkspacerouting.controller.devfile.io"
```

Comment:
:warning: It means old DevWorkspaces with such finalizers will hangs in terminating state, but I think we're safe since it's only OpenShift OAuth which uses it so far.

But we also can leave some safe double check if somebody thinks it worths it.

Reply:
One more example with finalizers - Che routing =(
Which mean there will be hanging workspaceroutings, after Che 7.27 -> 7.28 update.

The only safe option - let DWO for a few releases clean up workspacerouting CRs (Che DWO should listen to workspacerouting and devworkspacerouting...), then Che Operator can remove CRD.

Or just assume nobody used devworkspaces with Che 7.27.1 and just rename.